### DISCUSSION TITLE
Welcome to Data-Structures-and-Algorithms-Python Discussions!

### DISCUSSION THREAD

**Discussion by shushrutsharma** [shushrutsharma/Data-Structures-and-Algorithms-Python#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swhouse** [comment]
I appreciate what you have to share here. Thanks for sharing!
I have one question about Legacy directories. I noticed several folders like this: Structures/{Stacks,Queues,Trees,"Hash Tables","Linked Lists"}/Legacy. They look like an older version of DS implementation. What are the purposes of these folders?
